tools/codegen/api/unboxing.py

```diff
@@ -0,0 +1,199 @@
+from typing import List, Tuple
+
+from tools.codegen.api.cpp import argumenttype_type, argument
```

Comment:
See my previous comment here: https://github.com/pytorch/pytorch/pull/69881/#discussion_r794867257.

we have a bunch of `def argument()` functions floating around in the codegen under different API's, so you want to be forced to call `cpp.argument()` in this file, instead of just calling `argument()`. That way it's immediately clear which API is in use.